apps/portal/src/app/connect/wallet/sign-in-methods/external-wallets/page.mdx

```diff
@@ -0,0 +1,26 @@
+# External wallets
+
+thirdweb supports a number of first-party web3 wallets such as Metamask and Phantom. 
```

Comment:
"thirdweb natively supports 350+ first-party wallets and all wallets that implement EIP-6963 (Multi-injected Provider Discovery)"